include/jemalloc/jemalloc.sh

```diff
@@ -3,6 +3,8 @@
 objroot=$1
 
 cat <<EOF
+#pragma GCC system_header
+
```

Comment:
Move to after the `#define JEMALLOC_H_`. Worth making sure nothing happens when the header is already included.